### PR TITLE
7.1 Data Changes

### PR DESCRIPTION
**Pull request by Shinryin**
updates siren and turtle data
untested
siren extraction code needed some work, they changed element structure seems
a quick and dirty attempt at some working js that i used to make this update can be found here: https://gist.github.com/Shinryin/a622cad6a359cd56ba30d7220405bcfc
you can test and update the wiki
turtle seemed to still work fine :)

### CODE REVIEW

File: ScoutHelper/Data/Turtle.json [hunk 1]
```diff
@@ -1,24 +1,24 @@
 ﻿{
-	"arr": {
+	"arr" : {
```
Comment (dit-zy) [inline ScoutHelper/Data/Turtle.json]: oof, can you remove these extra spaces?

File: ScoutHelper/Data/Turtle.json [hunk 2]
```diff
@@ -1217,7 +1217,6 @@
 				"id": 1191,
 				"points": {
 					"905": {"x": " 8.50", "y": "33.30"},
-					"906": {"x": "12.80", "y": "21.40"},
```
Comment (dit-zy) [inline ScoutHelper/Data/Turtle.json:1220]: hmmm how strange. i wonder what happened to this point

Comment (Shinryin) [reply ScoutHelper/Data/Turtle.json:1220]: that's quite strange. good question
relating to this and your comment above, it was straight copy pasted from running the wiki's turtle code, so i'll take a look there for both issues as well as review the site real quick

File: ScoutHelper/Data/Turtle.json (hunk 1 above)
Comment (Shinryin) [reply ScoutHelper/Data/Turtle.json]: done
very simple change in wiki code
```
return `"${mob.name.toLowerCase()}" : ${mob.id}`
return `"${patch.name.toLowerCase()}" : {
```

extra spaces before the : on those lines needed to be removed

**Review by dit-zy — APPROVED** [review]
approved!

**Pull request merged**